CONTRIBUTING.adoc

```diff
@@ -86,13 +90,10 @@ export PATH=$PATH:$GOPATH/bin
         $ cd _output/local/bin/linux/amd64
         $ sudo ./openshift start
 
-+
-NOTE: Replace "linux/amd64" with the appropriate value for your platform/architecture.
-
 10.  Launch another terminal, change into the same directory you started OpenShift, and deploy the private docker registry within OpenShift with the following commands (note, the --credentials option allows secure communication between the internal OpenShift Docker registry and the OpenShift server, and the --config option provides your identity (in this case, cluster-admin) to the OpenShift server):
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
-        $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
+        $ sudo chmod a+rw openshift.local.config/master/admin.kubeconfig
```

Comment:
We do that in a few places:

images/origin/Dockerfile:
ENV KUBECONFIG /var/lib/openshift/openshift.local.config/master/admin.kubeconfig

vagrant-openshift/action/install_openshift.rb:
export KUBECONFIG=/openshift.local.config/master/admin.kubeconfig

contrib/vagrant/provision-util.sh:
local path="${config_root}/openshift.local.config/master/admin.kubeconfig"
echo "export KUBECONFIG=${path}" >> "${target}"

The installer method of copying the generated kubeconfig to a home directory could work, but would require generating the config first, which vagrant isn't doing at the moment.

I am not sure of all the things expecting to log into vagrant images and immediately have cluster admin permissions. @danmcp, any guidance on this?